### PR TITLE
optimize: optimize tcc module code handling

### LINKED ISSUE #3690: `Optimize Request:` Improve code quality.（提升代码质量请求）
Use the `SonarLint` plugin, `FindBugs` plugin or other plugin to analysis the `seata` project.
There will be a lot of code quality problems, even bugs. I hope people who are interested can help optimize the code.

使用`SonarLint`插件、`FindBugs`插件或其他插件来分析`seata`项目源码。
会找出非常多的代码质量问题，甚至有可能是隐藏的BUG。希望有兴趣的同学能够帮助提升代码质量。
在seata的源码中找找茬，提PR参与社区共建。
非常感谢。

# the analysis report like this（分析报告如下）:
图片

### PR DESCRIPTION
**Pull request by pengten**
- [x] I have registered the PR [changes](https://github.com/seata/seata/tree/develop/changes).

### Ⅰ. Describe what this PR did
optimize tcc module code handling

### Ⅱ. Does this pull request fix one issue?

fixes #3690

### CODE REVIEW

File: changes/en-us/1.5.0.md [hunk 1]
```diff
@@ -149,6 +149,7 @@
   - [[#4212](https://github.com/seata/seata/pull/4212)] optimize the interface of the console
   - [[#4216](https://github.com/seata/seata/pull/4216)] no more attempt to clean undolog for none AT user
   - [[#4251](https://github.com/seata/seata/pull/4251)] optimize partial code handling
+  - [[#4262](https://github.com/seata/seata/pull/4262)] optimize：optimize tcc module code handling
```
Comment (wangliang181230) [inline changes/en-us/1.5.0.md:152]: `optimize：` 去掉吧，有点多余。

**Comment by wangliang181230** [comment]
多余的import移除一下。
图片

**Review by wangliang181230 — APPROVED** [review]
LGTM

**Review by slievrly — APPROVED** [review]
LGTM

**Pull request merged**